finalPrep/README.md

```diff
@@ -18,37 +18,43 @@ Some things to consider as you refactor include...
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [X] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [X] Study/Project work time
+- [X] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1440" alt="Screen Shot 2021-08-07 at 6 43 05 PM" src="https://user-images.githubusercontent.com/78453792/128618929-55288abf-f61d-44d1-9c2a-f878825de997.png">
+
+- `Add Week 2 Screenshot Here`<img width="1440" alt="Screen Shot 2021-08-07 at 6 43 09 PM" src="https://user-images.githubusercontent.com/78453792/128618945-422fbfeb-d498-4a1b-ab40-fdc6d71cef64.png">
+
+- `Add Week 3 Screenshot Here` <img width="1440" alt="Screen Shot 2021-08-07 at 6 43 13 PM" src="https://user-images.githubusercontent.com/78453792/128618955-c2ca1ede-0085-4655-a507-cb3f06172e65.png">
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] Add link to your gist here: <a href="https://gist.github.com/cagallo/e7a92a46409f78a4d81eb88d94d91f9c">Mentorship DTR Prep</a>
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [X] Add a link to your gist here: <a href="https://gist.github.com/cagallo/e1b2d4994a26fbca24b172a66de3fb94">Lesson Prep</a>
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [X] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [X] Add a link to your gist here: <a href="https://gist.github.com/cagallo/3325fb7b0ff8403ea93c5316e1e3d9b4">Group Work Prep</a>
```

Comment:
Nice self-awareness here- what you have mentioned with anxiety (just being real here) will certainly come into play at times at Turing, just given the deadlines, and how project work never seems finished. I encourage you to be open with group project partners and instructors about times you feel this impacting you, as you may miss out on support if these feelings stay bottled up. Speaking for myself, I at times realized I would have been better off processing my feelings to help regain my focus as opposed to telling myself to just keep pushing forward, then realizing 2 hours later I really didn't maintain focus during that time. Of course, there is a balance here, and Turing will be an opportunity to try a variety of strategies to ease the nerves, and stay focused.